packages/main/src/backend/configManager/configManager.ts

```diff
@@ -35,6 +36,23 @@ export async function updateConfig(configPath: string, configToUpdate: Config):
   await fs.writeFile(configPath, encryptedConfigStr);
 }
 
+export async function updateOtpLongTermToken(
+  key: CompanyTypes,
+  otpLongTermToken: string,
+  configPath: string = configFilePath,
+): Promise<void> {
+  const config = await getConfig(configPath);
+  const account = config.scraping.accountsToScrape.find((acc) => acc.key === key) as {
+    loginFields: { otpLongTermToken: string };
+  };
+  if (account) {
+    account.loginFields.otpLongTermToken = otpLongTermToken;
+    await updateConfig(configPath, config);
+  } else {
+    throw new Error(`Account with key ${key} not found`);
+  }
```

Comment:
return early: if not `account`, throw. Otherwise, continue.


```suggestion
  if (!account) {
    throw new Error(`Account with key ${key} not found`);
  }

  account.loginFields.otpLongTermToken = otpLongTermToken;
  await updateConfig(configPath, config);
```